src/types.h

```diff
@@ -233,10 +264,7 @@ SizedType CreateUInt32();
 SizedType CreateUInt64();
 
 SizedType CreateString(size_t size);
-SizedType CreateArray(size_t num_elem,
-                      Type elem_type,
-                      size_t elem_size,
-                      bool elem_is_signed);
+SizedType CreateArray(size_t, SizedType);
```

Comment:
Style nit:
```suggestion
SizedType CreateArray(size_t num_elements, SizedType element_type);
```
At a glance this looks like a single parameter type `size_t` with name `SizedType`. Also the API is less obvious if I just read the header